### ISSUE TITLE
Bug: specials are not working

### ISSUE THREAD

**Issue by Barkole** [depcheck/depcheck#551]
In my project https://gitlab.com/Lepardo/simplewebspider-nodejs I enabled these specials:

```yaml
specials:
  - "bin"
  - "eslint"
  - "husky"
  - "jest"
  - "lint-staged"
  - "prettier"
```

But still all the eslint etc related dependencies are reported:

```txt
$ yarn depcheck --specials=bin,eslint,husky,jest,lint-staged,prettier --ignores=""
Unused devDependencies
* @typescript-eslint/eslint-plugin
* @typescript-eslint/parser
* eslint-config-prettier
* eslint-config-standard
* eslint-plugin-import
* eslint-plugin-node
* eslint-plugin-prettier
* eslint-plugin-promise
* eslint-plugin-standard
* prettier
* retire
* typescript
```
